pypeit/spectrographs/soar_goodman.py

```diff
@@ -335,9 +335,15 @@ def config_specific_par(self, scifile, inp_par=None):
         par = super().config_specific_par(scifile, inp_par=inp_par)
 
         # Wavelength calibrations
+        # Here is a useful website with an arc atlas
+        # http://soartelescope.org/soar/content/goodman-comparison-lamps
         if self.get_meta_value(scifile, 'dispname') == '400_SYZY':
             par['calibrations']['wavelengths']['reid_arxiv'] = 'soar_goodman_red_400_SYZY.fits'
             par['calibrations']['wavelengths']['method'] = 'full_template'
+        elif self.get_meta_value(scifile, 'dispname') == '600_SYZY_OLD':
```

Comment:
is it really called `600_SYZY_OLD`  ?  :)

Reply:
It is!  :-D